Code/app/src/main/AndroidManifest.xml

```diff
@@ -5,18 +5,19 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.INTERNET" />
-
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 
     <application
-        android:name="ca.ualberta.moodroid.ContextGrabber"
```

Comment:
Why was this changed?

Reply:
I didn't change this file? Reject this part I guess